tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -216,3 +221,12 @@ function getSortableValue(value: number | string | undefined | null) {
   }
   return value;
 }
+
+function getSortableValue(point: SelectedStepRunData, header: ColumnHeaders) {
+  switch (header) {
+    case ColumnHeaders.RUN:
```

Comment:
Can you leave a comment explaining the reasoning?

Reply:
I've added a comment.